ios/TextilePhotos.xcodeproj/xcshareddata/xcschemes/TextilePhotos.xcscheme

```diff
@@ -81,7 +90,7 @@
       </AdditionalOptions>
    </TestAction>
    <LaunchAction
-      buildConfiguration = "Debug"
+      buildConfiguration = "Release"
```

Comment:
Did you change the scheme's 'run' action to `Release` and forget to change it back?

Reply:
so there are now two schemes. for the 'release' scheme, i guess you'd still want a debug build for logs etc?

Reply:
Oh, right. Let me take a step back here and make sure I understand. Why do we need a new scheme? Within a single scheme, can't we load a different set of env vars based on whether it's a debug or release build? Maybe I don't understand the way react-native-config works...

Reply:
I'm guessing that doesn't achieve some goal you had in mind, just want to understand what that is.

Reply:
I guess the way I'd think about the different schemes is not as 'release' and 'debug', but different environments, 'dev' and 'production'. You might want to run either scheme in debug or release mode depending on what you're doing. For that reason, I'd leave the 'run' action set to 'debug' so things behave as usual with Xcode development.

Reply:
cool, updated